src/main/components/RequestBar.jsx

```diff
@@ -60,26 +71,27 @@ const RequestBar = (props) => {
 
   return (
     <div>
-      <form onSubmit={sendFetch} >
+      <Form onSubmit={sendFetch} onInvalid={handleInvalid}>
         {
           (SourceOrDest === 'source')
-          && <select name='method' id='fetchTypeInput' multiple={false} value={selected}
+          && <Select name='method' id='fetchTypeInput' multiple={false} value={selected}
             onChange={handleChange} >
             <option value='GET'>GET</option>
-          </select>
+          </Select>
         }
         {
           (SourceOrDest === 'dest')
-          && <select name='method' id='fetchTypeInput' multiple={false} value={selected}
+          && <Select name='method' id='fetchTypeInput' multiple={false} value={selected}
```

Comment:
may also be for another PR but  the options seem to still have the default styling, we should revisit post MVP.